joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/utils/AstGenRunner.scala

```diff
@@ -6,26 +6,83 @@ import io.joern.jssrc2cpg.preprocessing.EjsPreprocessor
 import io.joern.x2cpg.SourceFiles
 import io.joern.x2cpg.utils.ExternalCommand
 import io.shiftleft.utils.IOUtils
+import com.typesafe.config.ConfigFactory
 import org.slf4j.LoggerFactory
+import versionsort.VersionHelper
 
 import java.nio.file.Paths
 import scala.util.Failure
 import scala.util.Success
 import scala.util.matching.Regex
 import scala.util.Try
+import scala.sys.process.stringToProcess
 
 object AstGenRunner {
+
+  private val logger = LoggerFactory.getLogger(getClass)
+
   case class AstGenRunnerResult(
     parsedFiles: List[(String, String)] = List.empty,
     skippedFiles: List[(String, String)] = List.empty
   )
+
+  private val ExecutableName = Environment.OperatingSystem match {
```

Comment:
style, here and in other places: the value name should be lowercase, i.e. `val executableName`. Only constants are uppercase by convention, e.g. `val MagicNumber = 42`

Reply:
Right, will change that 👍